edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/TwinManager.cs

```diff
@@ -103,6 +103,36 @@ public async Task UpdateReportedPropertiesAsync(string id, IMessage reportedProp
 
         internal static void ValidateTwinProperties(JToken properties) => ValidateTwinProperties(properties, 1);
 
+        // TODO: Move to a Twin helper class (along with Twin manager update).
```

Comment:
Nit: why was this method moved? VS?

Reply:
it moves up as static method has higher order than class method.  Resharper formatting rule.

Reply:
As I changed this file, I just ran format code by Resharper, which move this method.